docs/user-guide/deploying-application/rollback-deployment.md

```diff
@@ -0,0 +1,36 @@
+# Rollback Deployment
+
+
+Deployments can be rolled back manually. After a deployment is completed, you can manually rollback a stage to a previously deployed image by retaining the same configuration or changing the configuration.
```

Comment:
Deployments can be rolled back manually. After a deployment is completed, you can manually rollback ~~a stage~~ to a previously deployed image by retaining the same configuration or changing the configuration.

| Rollback Options      | Description |
| ----------- | ----------- |
| Old Image only      | Rollback to old image only, any configurations changes sent with the deployment will NOT be rolled back.       |
| Old Image + Configs deployed with that image      | Rollback to old image and to old configurations which were deployed with that image.       |

Reply:
Ignore

Reply:
Deployments can be rolled back manually. After a deployment is completed, you can manually rollback ~~a stage~~ to a previously deployed image by retaining the same configuration or changing the configuration.